### PR TITLE
Bump version to 2.6.0

### PR DESCRIPTION
**Pull request by yokaze**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.6.0] - 2021-05-25
+
+### Changed
+- Update etcdutil version to v1.4.0 (#217).
```
Comment (morimoto-cybozu) [inline CHANGELOG.md:11]: @yokaze 
The main purpose of #217 seems not to update etcdutil but to renew the supported version of etcd to 3.4.
The topic here would be that the users of sabakan now can/must use etcd 3.4, right?

Comment (yokaze) [reply CHANGELOG.md:11]: @morimoto-cybozu 
I think yes, the important thing is to make etcdutil compatible for etcd 3.4.
I did not find any breaking changes that affect sabakan, so it is just preferred to use etcd 3.4.

**Comment by umezawatakeshi** [comment]
I don't have anything in particular to comment.

File: CHANGELOG.md (hunk 1 above)
Comment (morimoto-cybozu) [reply CHANGELOG.md:11]: @yokaze 
Thanks for confirmation.
I suggest the following changelog.

```suggestion
- Support etcd 3.4 as a backend store in addtion to etcd 3.3 (#217).
```

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**